app/presenters/email_alert_presenter.rb

```diff
@@ -89,14 +102,34 @@ def document_noun
     }
   end
 
-  def subject
+  def title
     redrafted? ? document.title + " updated" : document.title
   end
 
+  def summary
+    document.summary
+  end
+
+  def change_note
+    document.change_note
+  end
+
+  def subject
+    title
+  end
+
   def urgent
     document.urgent
   end
 
+  def public_updated_at
+    document.public_updated_at
```

Comment:
Apparently this can still come through as `null`... https://sentry.io/govuk/app-email-alert-api/issues/391875467/